Cubical/Algebra/Group/Abelianization/Properties.agda

```diff
@@ -151,20 +144,19 @@ module _ (G : Group ℓ) where
       η ((inv (c · b)) · (inv a))              ≡⟨ cong η (sym (invDistr a (c · b))) ⟩
       η (inv (a · (c · b))) ∎)
 
-  assocAb : (x y z : Abelianization) → x ·Ab (y ·Ab z) ≡ (x ·Ab y) ·Ab z
+  assocAb : (x y z : Abelianization G) → x ·Ab (y ·Ab z) ≡ (x ·Ab y) ·Ab z
```

Comment:
Should now be named `·AbComm` rather then `assocAb` accordng to the new naming guideline ?

Reply:
With the requested changes, ```assocAb``` will become a local name - I don't think we should apply the naming conventions to local names, only to those that will be used later. The names defined here are packaged as an ```AbGroupStr```...

Reply:
This a good question. 
To me the point of the naming were to have uniform naming in files, even for local names.
Maybe it should precise in NamingAlgebra.

Reply:
I don't think we want a convention for local names, but that should also be discussed somewhere else.

Reply:
Yes sure, it can always be renamed once this has been decided.

Reply:
if the name should be changed, then to ·AbAssoc (as this is asscociativity and not commutativity). Since the name won't get exported i'll resolve the conversation though